### PR TITLE
secp256k1-tr: fix no-std support

### PR DESCRIPTION
**Pull request by conradoplg**
Also regenerates some .md files that were missing in #774

### CODE REVIEW

File: frost-secp256k1-tr/src/lib.rs [hunk 1]
```diff
@@ -10,6 +11,7 @@ extern crate alloc;
 use alloc::borrow::Cow;
 use alloc::borrow::ToOwned;
 use alloc::collections::BTreeMap;
+use alloc::vec;
 use alloc::vec::Vec;
```
Comment (StackOverflowExcept1on) [inline frost-secp256k1-tr/src/lib.rs:15]:
```suggestion
use alloc::vec::{self, Vec};
```

Comment (StackOverflowExcept1on) [reply frost-secp256k1-tr/src/lib.rs:15]: same for `use alloc::{Cow, Borrow}`

**Comment by StackOverflowExcept1on** [comment]
We could also apply the recent changes #767 to frost-secp256k1-tr.

**Comment by conradoplg** [comment]
> We could also apply the recent changes #767 to frost-secp256k1-tr.

Good call, added in b558f41d

**Review by natalieesk — APPROVED** [review]
Looks good

**Pull request merged**